### ISSUE TITLE
Embeded video in text block modul

### ISSUE THREAD

**Issue by kaxigt** [ThemeFuse/Unyson#1500]
I'm trying to embed a video via wp media uploader in the text block modul instead of using the specific video modul but it doesn't work. How to fix this?

**Comment by moldcraft** [comment]
Hello,

Sorry, I don't know, I am not a WP Editor expert.

Try to switch to text mode add the embed shortcode manually

```
[embed]https://www.youtube.com/embed/7mMLlg9ZCJM[/embed]
```

I will inspect this problem later.

**Comment by moldcraft** [comment]
Also you can switch to text mode and add

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/SOLOZRVNM8w" frameborder="0" allowfullscreen></iframe>
```

**Comment by pear1** [comment]
Hello,

Our clients also have complains about this issue.
They are trying to insert videos under tab shortcode container and it is not working.

Are there any news about this issue?

**Comment by moldcraft** [comment]
Sorry, no news.

**Comment by moldcraft** [comment]
I have tried **wp-editor > Add Media > Insert from URL > YouTube link**, it throws a js error and the preview is showing endless loading

![1476375467997](https://cloud.githubusercontent.com/assets/1475489/19357333/df3d2f98-9179-11e6-88d1-6508ed4ac872.jpg)

**Comment by andreiglingeanu** [comment]
@moldcraft It works for me and I'm not able to reproduce that.

**Comment by kaxigt** [comment]
Sooo, am I understand this right you won't do anything about it?

**Comment by andreiglingeanu** [comment]
Well it worked on my side. Have you tried it and it fails for you?

**Comment by kaxigt** [comment]
Yes I have tried it, and appearently others too without a good result. I tried both on child as parent standard theme.

**Comment by danyj** [comment]
@andreiglingeanu  

>Well it worked on my side.

sorry just had to :)

https://varianto25.com/stickers/works-on-my-machine

sorry @kaxigt , im sure it will be looked at

**Comment by andreiglingeanu** [comment]
@kaxigt Can you please make a short video out of your problem?

**Comment by kaxigt** [comment]
Now I have tried again. Embed video doesn't work in text block.

**Comment by andreiglingeanu** [comment]
You mean in frontend?

**Comment by warudin** [comment]
Is it me or does adding a YouTube url in a text-block still not work?

When adding a link in a text-block the back-end shows the video, when checking front-end, only the url of the video shows up. When putting `[embed]` tags around it, nothing shows.

**Comment by andreiglingeanu** [comment]
Have you tried to pass the text block contents through the `do_shortcode()` function?

**Comment by warudin** [comment]
I haven't changed the code in the text-block, so this is what happens, which is the default:

`<?php echo do_shortcode( $atts['text'] ); ?>`

**Comment by andreiglingeanu** [comment]
There is definitely some differences in the way these editors are implemented... The editor from the `Widgets -> Text` works expectedly, it properly inlines the Embed URL: ![](https://dzwonsemrish7.cloudfront.net/items/3K2l0u0X2B1I1V2R3k3S/Image%202018-07-18%20at%203.00.44%20PM.png?v=bdf6f250)

I'll try to dig this...

**Comment by danyj** [comment]
@andreiglingeanu   

this is what @warudin said 

> when checking front-end, only the url of the video shows up. 

so it means the do_shortcode frontend does not work ,  but we need to know if @warudin  is talking about builder shortcode or the sidebar

**Comment by andreiglingeanu** [comment]
I believe he's talking about the builder shortcode. I can reproduce this thing on my side too

**Comment by yura-x** [comment]
Has anyone fixed this issue?
I'm also have this issue. I'm only could show YouTube video in WP-editor by putting whole iframe tag in text mode of editor:
`<iframe width="560" height="315" src="https://www.youtube.com/embed/SOLOZRVNM8w" frameborder="0" allowfullscreen></iframe>`

**Comment by mhsohag11** [comment]
anyone here ??? How can I show a video in a FW block ( type : wp-editor )?
